package.json

```diff
@@ -960,6 +960,11 @@
                     "default": true,
                     "description": "Show warning dialog when uploading a document to the cloud."
                 },
+                "cosmosDB.batchSize": {
```

Comment:
Do we want this to be specific to Cosmos? I could see us using this batchSize for other database types in the future.

Reply:
I thought that too but didn't want to be misleading since it only applies to Cosmos atm. 

With that said it's probably worth making it a generic setting to future proof it